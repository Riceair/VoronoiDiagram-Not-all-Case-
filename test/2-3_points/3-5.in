#三點測試 鈍角三角形
3
398 93
233 263
345 197