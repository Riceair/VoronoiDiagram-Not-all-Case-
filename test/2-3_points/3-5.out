P 233 263
P 345 197
P 398 93
E -106918 -103901.6 182 48.43
E 182 48.43 41762 70608.43
E 182 48.43 65702 33438.43
